AmplifyPlugins/Storage/AWSS3StoragePlugin/Dependency/SdkTypealiases.swift

```diff
@@ -0,0 +1,16 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import AWSCognitoIdentity
+import AWSCognitoIdentityProvider
+import AWSClientRuntime
+import ClientRuntime
+
+public typealias SdkResult<R, E> = Result<R, SdkError<E>>
+
+public typealias NetworkResult = (Result<HttpResponse, Error>) -> Void
```

Comment:
Should we move this to AWSPluginsCore? I see that this is used in Auth plugin as well.

Reply:
I considered that.  I expect this will be temporary though and only needed until we fully migrate the plugins Swift concurrency, so decided to keep it plugin by plugin for now.  If we end up deciding to keep it permanently for some reason, I believe we should move it to AWSPluginsCore.

Reply:
I considered that.  This would require importing AWSPluginsCore wherever they're used, which would be unexpected.  Also, I expect this will be temporary and only needed until we fully migrate the plugins Swift concurrency, so decided to keep it plugin by plugin for now.  If we end up deciding to keep it permanently for some reason, we may want to reconsider this.